prospero-common/src/main/java/org/wildfly/prospero/actions/PrepareCandidateAction.java

```diff
@@ -50,20 +48,19 @@
 class PrepareCandidateAction implements AutoCloseable{
```

Comment:
Yes. The candidate is basically a newly provisioned server with the new feature pack. In next step that candidate is merged with existing server to apply the changes. This is a process that happened in Galleon under the hood, but we explicitly split it up in prospero to allow preparing candidate without stopping the server (in case of update/revert at least).